Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -262,29 +337,34 @@ - (void)viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id<UIVi
     [super viewWillTransitionToSize:size withTransitionCoordinator:coordinator];
     [coordinator animateAlongsideTransition:^(id < UIViewControllerTransitionCoordinatorContext > context) {
         [self updateInsetsForArticleViewController];
-        [self.previewController updatePreviewWithSizeChange:size];
     } completion:NULL];
 }
 
 - (void)updateInsetsForArticleViewController {
-    CGFloat topInset = [self.navigationController.navigationBar frame].size.height
-                       + [[UIApplication sharedApplication] statusBarFrame].size.height;
-
-    UIEdgeInsets adjustedInsets = UIEdgeInsetsMake(topInset,
-                                                   0.0,
-                                                   self.tabBarController.tabBar.frame.size.height,
-                                                   0.0);
+    // TODO: remove
+//    CGFloat topInset = [self.navigationController.navigationBar frame].size.height
+//                       + [[UIApplication sharedApplication] statusBarFrame].size.height;
+//
+//    UIEdgeInsets adjustedInsets = UIEdgeInsetsMake(topInset,
+//                                                   0.0,
+//                                                   self.tabBarController.tabBar.frame.size.height,
+//                                                   0.0);
+//
+//    self.articleViewController.tableView.contentInset          = adjustedInsets;
+//    self.articleViewController.tableView.scrollIndicatorInsets = adjustedInsets;
+//
+//    //adjust offset if we are at the top
+//    if (self.articleViewController.tableView.contentOffset.y <= 0) {
+//        self.articleViewController.tableView.contentOffset = CGPointMake(0, -topInset);
+//    }
+}
 
-    self.articleViewController.tableView.contentInset          = adjustedInsets;
-    self.articleViewController.tableView.scrollIndicatorInsets = adjustedInsets;
+#pragma mark - Toolbar Actions
 
-    //adjust offset if we are at the top
-    if (self.articleViewController.tableView.contentOffset.y <= 0) {
-        self.articleViewController.tableView.contentOffset = CGPointMake(0, -topInset);
-    }
+- (void)didTapRefresh {
```

Comment:
Unneeded method. We are using a block callback for the button, so no "action method" is needed - you should just call `[self fetchArticle]` directly from the block callback.